sphinx/util/pycompat.py

```diff
@@ -103,3 +103,40 @@ def execfile_(filepath, _globals):
         else:
             raise
     exec_(code, _globals)
+
+# ------------------------------------------------------------------------------
+# Internal module backwards-compatibility
+
+import warnings
+
+from six import class_types
```

Comment:
It seems some people used to use it. [[1]](https://bitbucket.org/bwanamarko/sphinxcontrib-matlabdomain/pull-request/1/enh-remove-unused-import-of-base_exception/diff) [[2]](https://github.com/CybOXProject/python-cybox/blob/645c419/cybox/utils/autoentity.py#L3) [[3]](https://github.com/earthreader/libearth/blob/bc9a25898e/docs/conf.py#L39)

If we conclude that's never been part of the public API we should also remove `base_exception` I guess.